app/src/data/settings_4px.ts

```diff
@@ -1,269 +1,290 @@
-import { Settings } from '@epam/uui';
-
-const settings_4px: Settings = {
-    sizes: {
-        defaults: {
-            alert: '48',
-            badge: '24',
-            button: '40',
-            checkbox: '24',
-            countIndicator: '24',
-            flexRow: '40',
-            filtersPanel: '40',
-            linkButton: '40',
-            numericInput: '40',
-            radioInput: '24',
-            richTextView: '14',
-            switch: '24',
-            tabButton: '48',
-            tag: '40',
-            text: '40',
-            textArea: '40',
-            textInput: '40',
-            rangeDatePicker: '40',
-            datePicker: '40',
-            labeledInput: '40',
-            statusIndicator: '40',
-            paginator: '32',
-            rating: '24',
-        },
-        alert: {
-            action: {
+const settings_4px = {
+    alert: {
+        sizes: {
+            default: '48',
+            actionMap: {
                 32: '24',
                 48: '32',
             },
         },
-        notificationCard: {
-            action: '32',
-        },
-        tag: {
-            countIndicator: {
+    },
+    badge: {
+        sizes: {
+            default: '24',
+            countIndicatorMap: {
+                24: '20',
                 32: '20',
                 40: '24',
-                48: '24',
             },
         },
-        pickerInput: {
-            toggler: {
-                defaults: {
-                    size: '40',
-                    tag: '40',
+    },
+    button: {
+        sizes: {
+            default: '40',
+        },
+    },
+    checkbox: {
+        sizes: {
+            default: '24',
+        },
+    },
+    countIndicator: {
+        sizes: {
+            default: '24',
+        },
+    },
+    dataTable: {
+        sizes: {
+            header: {
+                defaultRow: '40',
+                defaultCell: '40',
+                defaultResizeMarker: '12',
+                defaultCellPadding: '12',
+                defaultCellPaddingEdge: '24',
+                cellCheckboxMap: {
+                    40: '24',
+                    48: '24',
+                    60: '24',
                 },
-                tag: {
+                captionFontSize: '14',
+                captionFontWeight: '400',
+                captionSize: '20',
+                captionUppercase: '12',
```

Comment:
Probably can be moved to the css?
captionFontSize: '14',
captionFontWeight: '400',
captionSize: '20',
 captionUppercase: '12',